frontend/src/app/components/scan-viewer/scan-viewer.component.ts

```diff
@@ -16,8 +16,8 @@ import {LabelTag} from '../../model/labels/LabelTag';
 })
 export class ScanViewerComponent implements OnInit, AfterViewInit {
 
-    downloadingScanInProgress = false;
-    downloadingSlicesInProgress = false;
+    downloadingScanInProgress = true;
```

Comment:
Why are you changing this ?

Reply:
Scan Viewer initially doesn’t show the loading spinner, which very often leads to a race-condition in E2E tests which wants to click on the canvas. It looks like this:
1. Marker show up without any spinner.
2. Canvas is available.
3. E2E Tests clicks on the canvas layer.
4. Component logic requests images from the backend.
5. Spinner shows up.
6. E2E Tests cannot click on the marker any longer...

The solution enables spinner in the first step.